src/CardContent/Infographic/Infographic.types.ts

```diff
@@ -0,0 +1,14 @@
+import type { ReactElement } from "react";
+import type { VersionChipProps } from "../../VersionChip/VersionChip.types";
+
+export type InfographicProps = {
+  /**
+   * An alias for image property. Available only with media
+   * components. Media components: video, audio, picture, iframe, img.
+   */
+  media: string;
+  /**
+   * Optional version chip component.
+   */
+  versionChip?: ReactElement<VersionChipProps>;
```

Comment:
I don think this props is correct. What we need is a `version: string` prop and use it to decide inside the `infographic` component whether the `VershionChip` to be shown or not.